cmd/promtool/main.go

```diff
@@ -74,6 +74,7 @@ func main() {
 	queryRangeExpr := queryRangeCmd.Arg("expr", "PromQL query expression.").Required().String()
 	queryRangeBegin := queryRangeCmd.Flag("start", "Query range start time (RFC3339 or Unix timestamp).").String()
 	queryRangeEnd := queryRangeCmd.Flag("end", "Query range end time (RFC3339 or Unix timestamp).").String()
+	queryRangeStep := queryRangeCmd.Flag("step", "Query step size (duration - specify unit).").Duration()
```

Comment:
(nit) I'd remove ` - specify unit`.

Reply:
@simonpasquier sure - I waffled on that one.